.eslintrc

```diff
@@ -0,0 +1,5 @@
+extends:
+  - lennym
+
+globals:
+  fetch: readonly
```

Comment:
If it's not talking downstream (and it shouldn't be) then this won't be required.